assets/javascripts/discourse/initializers/init-code-review.js.es6

```diff
@@ -3,20 +3,23 @@ import { ajax } from "discourse/lib/ajax";
 import { popupAjaxError } from "discourse/lib/ajax-error";
 import DiscourseURL from "discourse/lib/url";
 import { findAll } from "discourse/models/login-method";
-
-function actOnCommit(topic, action) {
-  return ajax(`/code-review/${action}.json`, {
-    type: "POST",
-    data: { topic_id: topic.id },
-  })
-    .then((result) => {
-      if (result.next_topic_url) {
-        DiscourseURL.routeTo(result.next_topic_url);
-      } else {
-        DiscourseURL.routeTo("/");
-      }
-    })
-    .catch(popupAjaxError);
+import { computed } from "@ember/object";
+
+async function actOnCommit(topic, action) {
+  try {
+    let result = ajax(`/code-review/${action}.json`, {
```

Comment:
(Possible noob question) - do we need an `await` here? Otherwise, `result` will be a promise, not the result of the promise?

Reply:
Oh oh, good catch!